Game/src/Menu.java

```diff
@@ -1,121 +1,284 @@
+import javafx.fxml.FXMLLoader;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.Node;
+import javafx.scene.Parent;
 import javafx.scene.Scene;
 import javafx.scene.control.Button;
 import javafx.scene.control.Label;
+import javafx.scene.effect.DropShadow;
 import javafx.scene.image.Image;
 import javafx.scene.image.ImageView;
+import javafx.scene.input.KeyCode;
 import javafx.scene.layout.Pane;
 import javafx.scene.layout.StackPane;
+import javafx.scene.layout.VBox;
 import javafx.scene.paint.Color;
-import javafx.scene.text.Font;
-import javafx.scene.text.Text;
+import javafx.scene.shape.Rectangle;
+
+import java.io.IOException;
 
 public class Menu {
 
-    public static Scene menu(){
+    @SuppressWarnings("Duplicates")
```

Comment:
Be very very careful with `@SuppressWarnings`. It can be dangerous to turn off warnings without good reason. Asking @hikari1734 for opinion

Reply:
Yeah don't ever suppress warnings. With what we are doing, if you are suppressing warnings then there is a better way to achieve the task.